nimpy.nim

```diff
@@ -237,7 +237,12 @@ proc updateStackBottom() {.inline.} =
 proc pythonException(e: ref Exception): PPyObject =
   let err = pyLib.PyErr_NewException(cstring("nimpy" & "." & $(e.name)), pyLib.NimPyException, nil)
   decRef err
-  pyLib.PyErr_SetString(err, cstring("Unexpected error encountered: " & e.msg))
+  let errMsg: cstring =
```

Comment:
Ah, I didn't had any issues on arc/orc and didn't think about that gc collection since I am still a bit new to nims memory models). I use a string now and force-pushed. Thanks!